pandas/src/parser/tokenizer.h

```diff
@@ -124,6 +124,8 @@ typedef enum {
     EAT_LINE_COMMENT,
     WHITESPACE_LINE,
     SKIP_LINE,
+    QUOTE_IN_SKIP_LINE,
```

Comment:
at some point maybe document what these states actually mean a bit more (I know its mostly self-explanatory, but for future readers)